CAIPs/caip-27.md

```diff
@@ -57,7 +57,7 @@ The application would interface with an RPC provider to make request as follows:
 {
   "id": 1,
   "jsonrpc": "2.0",
-  "method": "provider_request",
+  "method": "wallet_requestMethod",
```

Comment:
I thought we preferred `wallet_invokeMethod`?

Reply:
That's correct! This has been updated